events-api/src/modules/events/events.repository.ts

```diff
@@ -18,4 +21,156 @@ export class EventsRepository {
 
     return result;
   }
+
+  static async getOne(eventId: string): Promise<EventModel[]> {
+    const result = db.execute<EventModel>(
+      `
+        SELECT
+            event_id as eventId,
+            event_name as eventName,
+            event_description as eventDescription,
+            event_location as eventLocation,
+            event_date as eventDate,
+            event_created_at as eventCreatedAt,
+            event_updated_at as eventUpdatedAt
+        FROM
+            events
+        WHERE event_id = ?`,
+      [eventId]
+    );
+
+    return result;
+  }
+
+  static async create(eventData: NewEvent): Promise<EventModel[]> {
```

Comment:
Method is called create, but array is returned. Seems a bit contradictory for me